eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -85,13 +85,19 @@ stages:
                         condition: and(succeeded(), ne(variables['Skip.AutoAddTag'], 'true'))
                         displayName: 'Set Tag and Additional Tag'
 
+                      - script: |
+                          npm install $(Package.Archive)
+                        displayName: Install package
```

Comment:
Lets change this name to indicate its purpose a little further. Perhaps call it "Validating package can be installed"